<!DOCTYPE HTML>
<!--
Massively by HTML5 UP
html5up.net | @ajlkn
Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
<head>


	<title>Research</title>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<link rel="stylesheet" href="assets/css/main.css" />
	<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
</head>
<body class="is-preload">

	<!-- Wrapper -->
	<div id="wrapper">

		<header id="header">
			<a href="index.html" class="logo">JLM Lab</a>
		</header>

		<!-- Nav -->
		<nav id="nav">
			<ul class="links">
				<li><a href="index.html">About</a></li>
				<li class="active"><a href="research.html">Research</a></li>
				<li><a href="publications.html">Publications</a></li>
				<li><a href="people.html">People</a></li>
				<li><a href="join.html">Join Us</a></li>
			</ul>
			<ul class="icons">
				<li><a href="mailto:jleebensmack@uga.edu" class="icon fa-envelope"><span class="label">Email</span></a></li>
				<li><a href="https://twitter.com/jlmlab" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
				<li><a href="https://scholar.google.com/citations?user=etMU7ZQAAAAJ&hl=en" class="icon fa-google"><span class="label">Google Scholar</span></a></li>
				<li><a href="https://www.github.com/JLMLAB/" class="icon fa-github"><span class="label">GitHub</span></a></li>
			</ul>
		</nav>


		<!-- Main -->
    <div id="main">

			<!-- Post -->
			<section class="post">
				<header class="major">
					<h2>Research</h2>
				</header>
<div class="row">
				<header class="minor">
					<h3>Phylogenomic Investigations of Green Plant (Viridiplantae) Diversification</h3>
				</header>
				<p><span class="image right"><img src="https://static.wixstatic.com/media/eadafa_a416acb1fd7c4f328eed45a7d679e3a2~mv2.jpg/v1/fill/w_633,h_419,al_c,lg_1,q_80,enc_auto/greenplantdiversification.jpg" alt="" /></span>In collaboration with a large community of plant scientists involved in the 1000 Plant Transcriptomes Project, we have been generating and analyzing RNA seq data for over 1000 species distributed across all domains of green plant tree of life.  This work is helping us understand relationships among major plant lineages and elucidating the timing of whole genome duplications and large-scale gene family expansions.  Periods of rapid speciation evidenced in our inferred species phylogenies are marked by substantial gene tree - species tree discordance.  We are now working to gain deeper understanding of the processes contributing to this discordance and its consequences for comparative biology.  Working with the Department of Energy's Joint Genome Institute and many others in the plant genomics community, the Open Green Genomes initiative is generating reference quality genome assemblies and annotations for 35 species in pivotal positions across the land plant phylogeny.  Comparative analyses of these genomes will enable inferences about ancestral genome content and genomic changes associated with key innovations in land plant evolution.</p>
</div>
<div class="row">
				<header class="minor">
					<h3>Origin and Early Diversification of Flowering Plants</h3>
				</header>
				<p><span class="image left"><img src="https://static.wixstatic.com/media/eadafa_8a94a8ef69e8428686089e9ec719a3dc~mv2_d_3693_2638_s_4_2.jpg/v1/fill/w_637,h_455,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/amborellaMale1.jpg" alt="" /></span>Phylogenomic analyses have identified Amborella as the sister lineage to all other extant flowering plants.   The Amborella Genome Project has generated a genome assembly and annotation of the genome.  Analyses of gene content and gene order in the genomes of Amborella and other flowering plants is elucidating characteristics of the ancestral angiosperm genome including gene content and genome structure.  Interestingly, whereas gene order in the Amborella genome seems to have changed little over approximately 150 million years of evolution, the genome includes recently derived sex chromosomes.  This observation underscores the fact that the Amborella is the sister lineage to all other flowering plant lineages, but it is not the ancestral flowering plant species.</p>
</div>
<div class="row">
				<header class="minor">
					<h3>Monocot Systematics and Evolution</h3>
				</header>
				<p><span class="image right"><img src="https://static.wixstatic.com/media/eadafa_e04c2554c4114212830b7006c9dc2509~mv2_d_6016_3384_s_4_2.jpg/v1/fill/w_653,h_368,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/DSC_1247.jpg" alt="" /></span>The monocots are a diverse group including some 65,000 species and two of the most species rich plant families (Orchidaceae and Poaceae). Nearly all of what we know about monocot genomes is limited to economically important cereals within the grass family. While the vast amount genomic data for cereal species is quite valuable, recent work has shown that many genomic features of the grass family including GC content, codon usage, gene copy numbers and the identity and distribution of repetitive elements are distinct. Comparative analyses of grass and non-grass chloroplast genomes implicate rapid change in genome structure, substitution rates and codon usage biases occurred within the Poales but sometime before the radiation of major grass lineages at least 55 million years ago. An understanding of the processes that generated unique genome characteristics in the grasses will require whole genome comparative analyses including non-grass monocots. Moreover, an understanding of ecological and genetic events associate with rapid radiations in monocot history will require resolution of uncertainty in the relationships among major monocot lineages. Much of our research is aimed at understanding these processes through phylogenetic analyses of organismal relationships and comparative analyses of gene and genomes across the monocot phylogeny including pineapple, palms, banana, onion, asparagus, agave, yucca, orchids and  Acorus, the lineage to all other monocots.</p>
</div>
<div class="row">
				<header class="minor">
					<h3>Plant-Pollinator Coevolution</h3>
				</header>
				<p><span class="image left"><img src="https://static.wixstatic.com/media/eadafa_8a94a8ef69e8428686089e9ec719a3dc~mv2_d_3693_2638_s_4_2.jpg/v1/fill/w_637,h_455,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/amborellaMale1.jpg" alt="" /></span>Since my dissertation work on pollinator behavior in hybridizing Baptisia populations and my postdoctoral research with Olle Pelmyr on the yucca-yucca moth pollination mutualism I have been investigating various aspects plant-pollinator evolutionary ecology. My lab continues to explore diversification and the evolution of plant - insect interactions using the yucca - yucca moth system as a model. The thrust of this research is aimed at understanding the proximal mechanisms for pollinator host specificity and the evolution of reproductive isolation between yucca species. Variation in floral fragrance and flowering time both within and among species is influencing patterns of inter- and intraspecific gene flow.  Former student, Jeremy Rentsch, investigated genetic structure within and among yucca species of the southerstern U.S.. Jeremy's research revealed that Y. gloriosa is a hybrid species (Rentsch & Leebens-Mack 2012) derived from the C3 species Y. filamentosa and a CAM species, Y aloifolia.  Jeremy also showed that some Y aloifolia populations have recently opted out of yucca pollination and now rely on european honey bees as pollen vectors (Rentsch & Leebens-Mack 2014).</p>
</div>
<div class="row">
				<header class="minor">
					<h3>Crassulacean Acid Metabolism and its impact on biodiversity</h3>
				</header>
				<p><span class="image right"><img src="https://static.wixstatic.com/media/eadafa_8a94a8ef69e8428686089e9ec719a3dc~mv2_d_3693_2638_s_4_2.jpg/v1/fill/w_637,h_455,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/amborellaMale1.jpg" alt="" /></span>Building on Jeremy's work, Karolina Heyduk is developing Y. gloriosa as a model system for understanding the evolution of CAM photosynthesis within the Agavaceae.  Photosynthesis is a fundamental biological process supporting the vast majority of life on Earth.  For plants living under water-limited conditions, however, photosynthetic productivity can be greatly reduced by hotter and drier climatic conditions. To counteract these conditions, some plants utilize forms of photosynthesis that increase the efficiency with which they use water. One such innovation seen in plants growing in deserts or other water-limited habitats is Crassulacean Acid Metabolism or CAM.  CAM has evolved independently across diverse plant lineages and it is typically associated with stem (e.g. cacti) or leaf (e.g. agaves, some orchids) succulence. Working with collaborators at UC Riverside and the University of Buffalo we are integrating ecophysiological, genomics and evolutionary approaches to address fundamental questions about how plants use CAM, how genes involved in performing CAM are regulated in response to varying environmental conditions, and how the evolution of CAM facilitates diversification in water-limited habitats. To achieve these aims, we are investigating independent evolutionary transitions between typical C3 photosynthesis and CAM in the orchid and agave plant families, both of which have species known for their ability to thrive in water-limited environments. This research will provide a foundation for understanding the genetic basis of CAM pathways and contribute to ongoing efforts introduce CAM to economically important plants for improved water use efficiency when growing under drought conditions.</p>
</div>

			</section>


					</div>




				<!-- Copyright -->
				<div id="copyright">
					<ul><li>&copy; Jim Leebens-Mack</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
				</div>

			</div>

			<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

		</body>
		</html>
